### ISSUE TITLE
VS Tools->Options settings for using statements should move into Code Style section

### ISSUE THREAD

**Issue by jimmylewis** [dotnet/roslyn#33633]
**Version Used**: VS2019 Preview 4 (dogfood)

**Steps to Reproduce**:

1. Open Tools -> Options, find C# Code Style settings
2. Look for the settings about Using statements (they are actually located on the C# Advanced page)
  * Place 'System' directives first when sorting usings
  * Separate using directive groups
3. Set them as desired,
4. Export .editorconfig settings from the Code Style page

**Expected Behavior**:
These settings should be included in the generated .editorconfig file, since there are .editorconfig settings for them.

**Actual Behavior**:
They are left out, because they are not located in the Code Style page.  They *are* code style features though.

**Comment by jimmylewis** [comment]
This is more useful after #17658 is fixed.

**Comment by CyrusNajmabadi** [comment]
> They are left out, because they are not located in the Code Style page.

i don't see the connection here.  Nothing stops the generated .editorconfig from having these values, even if they're located elsewhere in the UI.

**Comment by jimmylewis** [comment]
Fair enough.  I guess to phrase it better, it seems like they are left out because they aren't on the same page, but they could still be part of the exported settings.

However, they are style settings, so it makes sense to me to move them here as well, especially to consolidate where all the settings are located.  The "Place System usings first" predates the Code Style settings page (which is probably why it remains on the Advanced page), but it is a style setting.

**Comment by sharwell** [comment]
Tentatively marking as **Won't Fix**, but will update with details and conclusion after we have a design review.

**Comment by jimmylewis** [comment]
@sharwell this setting isn't included in the exported .editorconfig either.  That seems like a more important issue than where the setting is visually located.  Can you consider this during that discussion?

**Comment by sharwell** [comment]
@jimmylewis that would be a normal bug that we can mark as **help wanted**. Can you file a separate issue for that?

**Comment by jimmylewis** [comment]
Opened #33766

**Comment by sharwell** [comment]
Design review conclusion: These options have been in the current location for a long time, and we don't want to disrupt users already familiar with this. We agree that the failure to export these was an unfortunate oversight and look forward to merging a fix for #33766.
